README.md

```diff
@@ -9,7 +9,7 @@ whatever mechanism or channel that is most appropriate for you. Once you have
 `jekyll` installed you should be able to view the site locally by running:
 
 ```bash
-jekyll run
+jekyll serve
```

Comment:
@thermokarst, `jekyll run` no longer shows up in the [docs](https://jekyllrb.com/docs/usage/). No need to preserve it for team members with older jekyll installs, right?